nucypher_ops/cli/main.py

```diff
@@ -7,8 +7,13 @@
 from nucypher_ops.cli.porter import cli as porter
 from nucypher_ops.cli.tbtcv2 import cli as tbtcv2
 
+from importlib.metadata import version
+
+package_version = version('nucypher_ops')
```

Comment:
Seems like the current version for `nucypher-ops` is wrong. There is a `0.12.0` tag, https://github.com/nucypher/nucypher-ops/tree/v0.12.0, yet the version number is `0.11.2`. 

Do you mind updating the version as part of this PR here - https://github.com/nucypher/nucypher-ops/blob/main/nucypher_ops/__about__.py#L7. Usually we would use bump version but since there is already a git tag, you can manually change it to `0.12.0`.

(cc @KPrasch )